.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Terminus Plugin Example
+on: push
+jobs:
+  checkout_lint:
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/pantheon-public/php-ci:v7.4
+    name: Checkout & lint
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Save repo content as artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: full-workspace
+          path: ${{ github.workspace }}
+      - name: Full Composer Install
+        run: composer install
+      - name: Install test tools
+        run: composer install-tools
+      - name: Validate Code
+        run: composer cs
+
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        php-versions: ['7.4', '8.0']
+    container:
+      image: pantheonpublic/terminus-plugin-test:4.x-php${{ matrix.php-versions }}
+    name: Test (PHP ${{ matrix.php-versions }})
+    needs: [ checkout_lint ]
+    defaults:
+      run:
+        shell: bash
+        working-directory: /terminus-plugin-test/plugin
+    env:
+      TERMINUS_TOKEN: ${{ secrets.TERMINUS_TOKEN }}
+      TERMINUS_SITE: ${{ secrets.TERMINUS_SITE }}
+      BASH_ENV: ~/.bashrc
+      HOME: /home/tester
+    steps:
+      - name: Download repo content from artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: full-workspace
+          path: /terminus-plugin-test/plugin
+      - name: Setup environment
+        run: chmod +x ./.ci/set-up-globals.sh && ./.ci/set-up-globals.sh
+      - name: Check Terminus version
+        run: terminus --version
+      - name: Install plugin
+        run: terminus self:plugin:install .
+      - name: Install SSH key
+        uses: shimataro/ssh-key-action@v2.3.1
```

Comment:
This is failing right now, presumably due to the lack of a secret. This plugin does not require a fixture site, so we have the deep philosophical question: should we set up testing fixtures for plugins that don't need them, just so that folks who copy the code have example test fixtures? There's some merit to that, but I'd say maybe we should just keep it simple and remove `TERMINUS_TOKEN` et. al. from these tests, and put them back in where needed.

Reply:
I'd comment stuff out in the ci.yml file. This way, it's still there for people to copy but it's not affecting this project. Maybe comment out the lines and add a comment that to enable people should add secret and uncomment?

Reply:
Will commenting make the tests pass though?  Can't we just make universal test secrets that are limited in scope to only testing repos?

Reply:
Commenting those sections in this specific plugin will make it pass because this plugin tests don't need those secrets

Reply:
The ssh-key-action is failing in GitHub actions.

Reply:
We could just set the secrets everywhere if we want the example here for reference.

Reply:
How do we plan on solving the ssh/secrets bottleneck?